Readme.MD

```diff
@@ -3,32 +3,67 @@
 
-## Privacy and Encryption
+### Checksums
 
-Mailchain considers privacy of account holders to be of utmost importance. All message fields will be encrypted to anyone except the recipient. The medium and location of the message is also encrypted.
+To check the application you downloaded matches the build SHA256 checksum.
 
-The Mailchain application will encrypt and decrypt message locations and messages locally on the client. No unintended, un-encrypted information will be transmitted at any time.
+For example, on a Mac:
 
-Sender & Recipient Privacy Considerations: It is not possible to obfuscate the sender of a message (or transaction sender) without compromising trust (i.e. passing some information to a 3rd party to ask them to send on behalf of another account). As a result, an observer can see that messages have been sent to a recipient and even watch for existence of 2-way conversations.
-Risks: Because no other information is revealed, this is considered no more of a risk than what happens when a normal transaction takes place.
+``` bash
+shasum -a 256 path_to_file/mailchain_...
+```
+
+The ouptut should match the checksum found in checksums.txt
 
 ## Spec
 
-### Chain Data
+See https://github.com/mailchain/mailchain-specification for the full requirments that Mailchain is built to address.
+
+### Blockchain Transaction Data
+
+Blockchains store transactions which act as notifications of a message being sent. The location of where the message is stored will be stored in the transaction data ('tx_data'). Each blockchain protocol may have differences in the way data is handled. Mailchain uses a standard format plus any protocol specifics.
 
-Chains store transactions which act as notifications of a message being sent and the location of where the message is stored. Each chain will have differences on the way the data is sent. Mailchain will use the same format plus chain specifics. *Note: In the case of Ethereum the data will be prefixed with 0x and added to the input field.* [MultiFormat](https://github.com/multiformats/multiformats) is a used to throughout Mailchain increase flexibility.
+*For example: In the case of Ethereum the data will be prefixed with 0x and added to the input field.* 
+
+An explaination of the Ethereum implementation can be count in [examples/ethereum_implementation.md](examples/ethereum_implementation.md) file.
 
 #### Format
 
-Chain data is formatted as follows `ChainData = encoded(ChainPrefix + Prefix + MultiFormatData + ChainSuffix)`. All values are stored are bytes that are then `encoded` to be stored on the chain. The `ChainPrefix` and `ChainSuffix` allow for any chain specific formatting to be applied.
+tx_data is formatted as follows `tx_data = encoded(ChainPrefix + Prefix + MultiFormatData + ChainSuffix)`. All values are stored are bytes that are then `encoded` to be stored on the chain. The `ChainPrefix` and `ChainSuffix` allow for any blockchain protocol-specific formatting to be applied.
```

Comment:
Thought we were getting rid of suffix?

Reply:
Yes.
Has this gone from the the rest of the repo?
Once it has, I'll remove refs everywhere:

[] this README
[] Ethereum Implementation
[] Mailchain specification